example-app/SantokuApp/src/framework/authentication/SecureStorageAdapter.ts

```diff
@@ -10,14 +10,14 @@ const STORED_ITEM_KEYS = {
 /**
```

Comment:
セキュリティ的な部分での設定理由は重要なので、コメントとして残すのはどうでしょうか？

```suggestion
// https://github.com/ws-4020/mobile-app-crib-notes/pull/640#issuecomment-984500781
const KEY_CHAIN_ACCESSIBILITY = SecureStore.WHEN_UNLOCKED;

/**
```